distribution/lib/Standard/Base/0.0.0-dev/src/Warning.enso

```diff
@@ -25,6 +25,12 @@ type Warning
         origin = Runtime.get_stack_trace
         attach_with_stacktrace value warning (origin.drop (Index_Sub_Range.First 1))
 
+    ## UNSTABLE
+
+       Are any warnings attached to the value?
+    has_warnings : Any -> Boolean
+    has_warnings value = has_warnings value
```

Comment:
I know this follows our name resolution conventions correctly, but wow that looks so much like it would infinite-loop.

Reply:
Maybe worth adding a comment inside of the method explaining this or just renaming the builtin to be safe? But please feel free to ignore, as it's not a priority thing.